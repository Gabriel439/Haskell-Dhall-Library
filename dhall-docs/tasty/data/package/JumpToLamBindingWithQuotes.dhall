{-|
This tests shows that the feature works for quoted vars on function parameters
as well
-}
let fun =
    λ(`param 1` : Text) ->
    λ(`param 2` : Text) ->
    [ `param 1` ++ `param 2` ]

let example0 = assert : fun "a" "b" === ["ab"]

in fun

