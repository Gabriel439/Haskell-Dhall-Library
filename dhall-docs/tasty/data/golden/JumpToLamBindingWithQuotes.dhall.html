<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/JumpToLamBindingWithQuotes.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >JumpToLamBindingWithQuotes.dhall</span></h2
      ><a data-path="/JumpToLamBindingWithQuotes.dhall"
          class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
        >This tests shows that the feature works for quoted vars on function parameters
as well</p>
</div
      ><h3
      >Examples</h3
      ><div class="source-code code-examples"
        ><pre
          >fun &quot;a&quot; &quot;b&quot; ≡ [ &quot;ab&quot; ]<br/></pre></div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          >{-|<br
          />This tests shows that the feature works for quoted vars on function parameters<br
          />as well<br
          />-}<br
          />let <span id="var5-5" data-variable="var5-5"
                      class="variable-decl"
          >fun</span
          > =<br
          />    λ(<span id="var6-7" data-variable="var6-7"
                        class="variable-decl"
          >`param 1`</span
          > : Text) -&gt;<br
          />    λ(<span id="var7-7" data-variable="var7-7"
                        class="variable-decl"
          >`param 2`</span
          > : Text) -&gt;<br
          />    [ <a href="#var6-7" data-variable="var6-7"
                     class="variable-use"
          >`param 1`</a
          > ++ <a href="#var7-7" data-variable="var7-7" class="variable-use"
          >`param 2`</a
          > ]<br
          /><br
          />let <span id="var10-5" data-variable="var10-5"
                      class="variable-decl"
          >example0</span
          > = assert : <a href="#var5-5" data-variable="var5-5"
                          class="variable-use"
          >fun</a
          > &quot;a&quot; &quot;b&quot; === [&quot;ab&quot;]<br
          /><br
          />in <a href="#var5-5" data-variable="var5-5" class="variable-use"
          >fun</a
          ><br
          /><br/></pre></div></div></body></html>